tests/Base.php

```diff
@@ -420,10 +420,13 @@ public function testLogs(): void
         self::assertStringContainsString('Error log', $errors);
         self::assertStringContainsString('Native log', $logs);
         self::assertStringContainsString('Native logs detected.', $logs);
-        self::assertStringContainsString('{"objectKey":"objectValue"}', $logs);
-        self::assertStringContainsString('["arrayValue"]', $logs);
+        self::assertStringContainsString('objectKey', $logs);
+        self::assertStringContainsString('objectValue', $logs);
+        self::assertStringContainsString('arrayValue', $logs);
```

Comment:
We are no longer checking for JSON object and array syntax because that's not important. Each language might print type, and other stuff. Format isn't important ,values are.